models/evc.py

```diff
@@ -1154,6 +1154,8 @@ def check_trace(
                     circuits_checked
                 ):
         """Auxiliar function to check an individual trace"""
+        if not circuit_current_path:
+            return False
```

Comment:
@gretelliz can you please check the case of intra-switch EVCs (i.e., both UNIs are in the same switch)? Also, please add unit tests to cover this change, just so we don't happen to decrease coverage!

Reply:
Added functions: `test_check_list_traces_same_switch_unis` and `test_check_list_traces_current_path_empty`

Reply:
Thank you, @gretelliz!! That functions didn't help much on the coverage (reducing missing terms) so I made a suggestion of removing those and adding just another check on commit dea502e. Please let me know if you are okay with that suggestion

Reply:
Also, I've documented in Issue #245 the problem of using sdntrace-cp to check the consistency for intra-switch EVCs. Apparently, sdntrace-cp don't help much the way it is currently. This should be addressed in another PR, IMHO.